cli/common/envoy/http.go

```diff
@@ -69,6 +73,53 @@ type Secret struct {
 	LastUpdated string
 }
 
+// CallLoggingEndpoint requests the logging endpoint from Envoy Admin Interface for a given port
+// more can be read about that endpoint https://www.envoyproxy.io/docs/envoy/latest/operations/admin#post--logging
+func CallLoggingEndpoint(ctx context.Context, portForward common.PortForwarder, params string) (map[string]string, error) {
```

Comment:
part of me thinks we should probably be validating the params when they come in, as of right now this is only called from the `cmd/proxy` package, but it could be called from elsewhere in the future and it would be nice to give feedback about bad formatting earlier in the process before we hit envoy. OTOH this could potentially be mitigated by having a struct that holds the params and parsing that into a string here

Reply:
ended up handling the validation in setting the logger params